esp8266/sonoffsc/sonoffsc.ino

```diff
@@ -19,7 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <Arduino.h>
-#include "config/all.h"
```

Comment:
This won't work in general. Are you adding the "config" to the included folders in your IDE, maybe?